statsmodels/robust/scale.py

```diff
@@ -49,6 +49,46 @@ def mad(a, c=Gaussian.ppf(3/4.), axis=0, center=np.median):
     return np.median((np.abs(a-center)) / c, axis=axis)
 
 
+def iqr(a, c=Gaussian.ppf(3/4) - Gaussian.ppf(1/4), axis=0, center=np.median):
+    # c \approx 1.349
```

Comment:
Perhaps remove below the docstring

Reply:
Removed in commit ce3770f2774fe3c61fd7b1e5c1d4657d3731e1bf